### DISCUSSION TITLE
Restore Firefox Sync

### DISCUSSION THREAD

**Discussion by yokoffing** [yokoffing/Betterfox#196] [Polls]
This poll is regarding restoring Firefox Sync. It is controlled by one pref in [Securefox](https://github.com/yokoffing/Betterfox/blob/443710b0738ebc8f45296a4f63c0bf362512d72f/Securefox.js#L1153-L1157):

```
// PREF: disable Firefox accounts
user_pref("identity.fxaccounts.enabled", false);
```

There was [recent feedback](https://news.ycombinator.com/item?id=36775054) saying that many users rely on Sync to keep tabs between mobile and desktop devices. I don't use Sync, but I understand the appeal.

Since Betterfox is aimed at semi-casual users all the way to enthusiasts, is this something you use? Should we leave this on by default?

Should Betterfox [user.js](https://github.com/yokoffing/Betterfox/blob/main/user.js) leave Firefox Sync enabled?

**Comment by trickypr** [comment]
Pulse overrides this setting because a few people asked for it. However, that doesn’t mean other betterfox users would want it enabled.

**Comment by Ryosuke-Asano** [comment]
I think you can do it freely.

 Floorp respects your judgment

**Reply by yokoffing** [reply]
@surapunoyousei Does Floorp enable Firefox Sync by default? I know Waterfox and Pulse does.

**Reply by Ryosuke-Asano** [reply]
Yeah. Enabled

**Comment by rokuuuuu** [comment]
is it possible to add a search feature when you open a pdf file?
